### PR TITLE
[angular-ui-router-uib-modal] remove typings provided by package directly

### PR DESCRIPTION
**Pull request by BendingBender**
One of the packages to remove from the list in #13364

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [x] Delete the package's directory.
- [x] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by BendingBender** [comment]
@nonplus: I am removing the typings for `angular-ui-router-uib-modal` from DefinitelyTyped because they are now available in the project repo (https://github.com/nonplus/angular-ui-router-uib-modal). Could you please give your ack?

**Comment by BendingBender** [comment]
@weswigham any idea on how to fix the build breakage?

**Comment by sheetalkamat** [comment]
@BendingBender Can you please fix the merge conflict. Thanks

**Comment by amcasey** [comment]
@nonplus, do you have any concerns about this change?

File: notNeededPackages.json [hunk 1]
```diff
@@ -19,6 +19,12 @@
             "asOfVersion": "0.0.5"
         },
         {
+            "libraryName": "angular-ui-router-uib-modal",
+            "typingsPackageName": "angular-ui-router-uib-modal",
+            "sourceRepoURL": "https://github.com/nonplus/angular-ui-router-uib-modal",
+            "asOfVersion": "0.11.0"
```
Comment (mhegazy) [inline notNeededPackages.json]: i think this should be `angular-ui-router-uib-modal@0.0.11` instead.

**Review by mhegazy — CHANGES_REQUESTED** [review]

File: notNeededPackages.json (hunk 1 above)
Comment (BendingBender) [inline notNeededPackages.json]: no idea how this happened, this was generated by dts-gen

**Pull request merged**